src/ExpressionEvaluator/Core/Source/ResultProvider/Expansion/MemberExpansion.cs

```diff
@@ -453,6 +459,36 @@ internal StaticMembersExpansion(Type runtimeType, Expansion members)
                 evalFlags: DkmEvaluationFlags.None);
         }
 
+        private static DkmClrCustomTypeInfo AddCustomModifiers(MemberAndDeclarationInfo member, DkmClrCustomTypeInfo typeDeclaringMemberInfo)
+        {
+            Type[] modopts, modreqs;
+            member.GetCustomModifiers(out modopts, out modreqs);
+
+            if (((modopts == null) || modopts.Length == 0) && ((modreqs == null) || (modreqs.Length == 0)))
+            {
+                // No modopts or modreqs.  Just return typeDeclaringMemberInfo as is.  This is the most common case.
+                return typeDeclaringMemberInfo;
+            }
+            else
+            {
+                // return a new DkmClrCustomTypeInfo object with the correct modopts/modreqs plumbed through.
+                var payloadTypeId = typeDeclaringMemberInfo != null ? typeDeclaringMemberInfo.PayloadTypeId : Guid.Empty;
+                var payload = typeDeclaringMemberInfo != null ? typeDeclaringMemberInfo.Payload : new ReadOnlyCollection<byte>(new byte[0]);
+                return DkmClrCustomTypeInfo.Create(payloadTypeId, payload, GetDkmClrTypeArray(modopts), GetDkmClrTypeArray(modreqs));
+            }
+        }
+
+        private static ReadOnlyCollection<DkmClrType> GetDkmClrTypeArray(Type[] lmrTypes)
```

Comment:
ReadOnlyCollection is because that's what the Concord API takes (the method he passes the return value to above).

RE:  "lemur" vs "lmr", it's consistently "lmr" throughout the EE code, so this should stay as-is.